server/routes/referrals/referralDataPresenterUtils.ts

```diff
@@ -54,4 +54,8 @@ export default class ReferralDataPresenterUtils {
       return aIndex - bIndex
     })
   }
+
+  static errorMessage(errors: { field: string; message: string }[] | null, field: string): string | null {
+    return errors?.find(error => error.field === field)?.message ?? null
```

Comment:
does this assume that the list of errors has a top-level 'field' parameter?

Reply:
Yup - that's what this bit of the method signature states:

```
errors: { field: string; message: string }[] | null
```

Reply:
right- i was confused that this object was the api response, but i see it's an object we create from the api response.

Reply:
Yeah, it'd be either an object resulting from a frontend validation error, _or_ something we create from an API error response.